index.js

```diff
@@ -168,11 +168,10 @@ function initApp(callback) {
         });
     });
 
-    app.get('/api/v1/substitution/run/:id/:image', (req, res) => {
+    app.get('/api/v1/substitution/run/:id', (req, res) => {
```

Comment:
The path `substutitution/run` is not part of the API draft (https://github.com/o2r-project/o2r-web-api/pull/45).
What is it for?